src/datalakebundle/table/schema/MetadataChecker.py

```diff
@@ -0,0 +1,90 @@
+import json
+from logging import Logger
+from pyspark.sql.session import SparkSession
+from pyspark.sql.utils import AnalysisException
+
+from datalakebundle.table.create.TableDefinition import TableDefinition
+from deepdiff import DeepDiff
+
+
+class MetadataChecker:
+    def __init__(self, logger: Logger, spark: SparkSession):
+        self.__logger = logger
+        self.__spark = spark
+
+    def check(self, table_definition: TableDefinition):
+        tables = self.__spark.sql(f"SHOW TABLES IN {table_definition.db_name}").toPandas()
+
+        if tables["tableName"].str.contains(table_definition.table_name).any():
+            self.__check_primary_key(table_definition)
+            self.__check_partition_by(table_definition)
+            self.__check_tbl_properties(table_definition)
+
+    def __check_primary_key(self, table_definition: TableDefinition):
+        pass
```

Comment:
Why empty?

Reply:
We haven't decided how we are going to check it. I just prepared the method for future. Should I delete it?

Reply:
I have an idea how to check it. We could use the table_properties to store the primary_key

Reply:
If it is not implemented, let's remove the code. This solution does not make any sense.